CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `pyrgg.py` renamed to `graph_gen.py`
 - Other functions moved to `functions.py`
 - Test system modified
+- Refactor `graph_gen.py` and `functions.py` using more Pythonic patterns.
```

Comment:
It would be better if you make your change logs atomic as you do in first PR comment, I mean something like this:
```
- `graph_gen.py` refactored
- `functions.py` refactored
```
```

Reply:
Thank you. I have just done this.